lib/lita/adapters/slack.rb

```diff
@@ -38,9 +38,28 @@ def roster(target)
         room_roster target.id, api
       end
 
-      def send_messages(target, strings)
+      # @param [Array] messages list of String messages or Symbol emoji reactions
```

Comment:
You dropped the emoji symbol part of this, this comment is outdated.

Reply:
Fixed.